src/main/java/com/dreammaster/scripts/ScriptAE2FC.java

```diff
@@ -664,6 +670,55 @@ public void loadRecipes() {
                 200,
                 500000,
                 true);
+        // Fluid Quantum Drive
+        GT_ModHandler.removeRecipeByOutput(AE2FC_QUANTUM_CELL);
```

Comment:
Please move recipe removals to the correct file (GT_Recipe_Remover or RecipeRemover)